h2o-genmodel/src/main/java/hex/genmodel/easy/error/VoidErrorConsumer.java

```diff
@@ -0,0 +1,15 @@
+package hex.genmodel.easy.error;
+
+import hex.genmodel.easy.EasyPredictModelWrapper;
+
+/**
+ * A void implementation of {@link hex.genmodel.easy.EasyPredictModelWrapper.ErrorConsumer}.
+ * It's purpose is to avoid forcing developers do to null checks in code before each and every call.
+ */
+public class VoidErrorConsumer implements EasyPredictModelWrapper.ErrorConsumer {
```

Comment:
The very existance of a default void error consumer ensures null checks are not forgotten.